benchmark/benchmark.py

```diff
@@ -0,0 +1,68 @@
+import pandas as pd
+from tabulate import tabulate
+import time
+import os
+import csv
+
+iterations = 100
+path = 'benchmark/'
+df = pd.read_csv('benchmark/Google_data_cleaned.csv')
+benchmark_result = {'File format': [], 'Size': [], 'Write time [ms]': [], 'Read time [ms]': []}
```

Comment:
please wrap this as a function

path and `'benchmark/Google_data_cleaned.csv'` should be parameters